test/e2e/framework/pod/resource.go

```diff
@@ -672,13 +672,13 @@ func GetPodsScheduled(masterNodes sets.String, pods *v1.PodList) (scheduledPods,
 		if !masterNodes.Has(pod.Spec.NodeName) {
 			if pod.Spec.NodeName != "" {
 				_, scheduledCondition := podutil.GetPodCondition(&pod.Status, v1.PodScheduled)
-				gomega.Expect(scheduledCondition != nil).To(gomega.Equal(true))
-				gomega.Expect(scheduledCondition.Status).To(gomega.Equal(v1.ConditionTrue))
+				framework.ExpectEqual(scheduledCondition != nil, true)
```

Comment:
The test jobs are failed like
```
# k8s.io/kubernetes/test/e2e/framework/pod
test/e2e/framework/pod/resource.go:675:5: undefined: framework
test/e2e/framework/pod/resource.go:676:5: undefined: framework
test/e2e/framework/pod/resource.go:680:5: undefined: framework
test/e2e/framework/pod/resource.go:681:5: undefined: framework
```
Because here is in the `framework` itself. We need to remove `framework` from here.

Reply:
Thx! I removed the` framework ` but it said ` undefined : ExpectEqual `. So I think we should import ` framework ` .  Now I am blocked by this ： https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/79771/pull-kubernetes-bazel-build/1147495162553831424
It just doesn't make sense - other packages are doing fine under the ` framework `.